lib/widgets/drive/drive-scanner.coffee

```diff
@@ -0,0 +1,27 @@
+compare = require('./compare')
+EventEmitter = require('events')
+_ = require('lodash')
+
+module.exports = class DriveScanner extends EventEmitter
+	constructor: (driveFinder,options) ->
+		super()
+		@drives = options.drives
```

Comment:
@jviotti Last one is already being [tested](https://github.com/resin-io/resin-cli-visuals/blob/modify-drive/tests/widgets/drive/drive-scanner.spec.coffee#L8)